backend.native/tests/framework/gh3343/objclib.m

```diff
@@ -0,0 +1,11 @@
+#include "objclib.h"
+
+NSString* foo1(int x) {
+  return [NSString stringWithFormat:@"%d", x];
+}
+
+NSArray* foo2(int x) {
+  NSValue* xx = @(x);
+  NSString* s = @"zzz";
+  return [NSArray arrayWithObjects: xx, s, nil];
+}
```

Comment:
You don't actually need an `.m` file compiled into dynamic library for top-level functions: placing these bodies to `.h` instead should be enough.